addons/loyalty/models/loyalty_program.py

```diff
@@ -50,7 +50,8 @@ class LoyaltyProgram(models.Model):
         ('next_order_coupons', 'Next Order Coupons')],
         default='promotion', required=True,
     )
-    date_to = fields.Date(string='Validity')
+    date_from = fields.Date(string="Start Date", help="The program is active including start date")
+    date_to = fields.Date(string="End date", help="The program is active including end date")
```

Comment:
Could you use another word than "active" here? It's a bit confusing IMO
Or maybe something more like: "The start date is included in the validity period of this program"